product_docs/docs/pem/8/pem_online_help/04_toc_pem_features/20_monitoring_a_pgd_cluster.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: "Monitoring a PGD Group"
+---
+
+<div id="monitoring_a_pgd_cluster" class="registered_link"></div>
+
+PGD (Bi-Directional Replication) is a PostgreSQL extension that provides a multi-master replicaton between the Postgres clusters. Before you monitor PGD through PEM console, you must ensure that your PGD nodes are up and running. For more information about PGD please visit the [EDB Doc Website](https://www.enterprisedb.com/docs/pgd/latest/).
```

Comment:
Does PGD stand for Bi-Directional Replication?? In any case, usually the full name is first and then the abbreviation.